robottelo/entities.py

```diff
@@ -2138,7 +2177,7 @@ class System(
         required=True,
     )
     # guest = entity_fields.OneToManyField()  # FIXME What does this field point to?
-    host_collection = entity_fields.OneToOneField('HostCollection')
+    host_collection = entity_fields.OneToManyField('HostCollection')
```

Comment:
When creating a system, you can specify zero or more host collections. When reading a system, you can specify only one host collection. Whatever POST accepts is the final word here.